nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -85,7 +85,11 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - The `nix.readOnlyStore` option has been renamed to `boot.readOnlyNixStore` to clarify that it configures the NixOS boot process, not the Nix daemon.
 
+<<<<<<< HEAD
```

Comment:
You forgot to resolve this merge conflict

Reply:
Ooof, thanks!